### PR TITLE
add v-hacd as an external project

### PR DESCRIPTION
**Pull request by birarda**

### CODE REVIEW

File: tools/vhacd/CMakeLists.txt [hunk 1]
```diff
@@ -1,23 +0,0 @@
...
-    find_package( Threads)
-    target_link_libraries(${TARGET_NAME} ${CMAKE_THREAD_LIBS_INIT})
-
-    include(FindOpenMP)
-    if(OPENMP_FOUND)
-        set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} ${OpenMP_C_FLAGS}")
-        set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${OpenMP_CXX_FLAGS}")
-        set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} ${OpenMP_EXE_LINKER_FLAGS}")
-   endif()
-endif()
```
Comment (birarda) [inline tools/vhacd/CMakeLists.txt]: Ended up removing this block in the renamed file - and the networking and octree links. My gut says neither is required but need to test expected functionality across platforms before this is merged.

**Comment by hifi-gustavo** [comment]
Build console output for commit: b25d22aa53858b2bf549b987883a03b00292e669 available at: https://gist.github.com/8366376bc638f2549670<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4650.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4650.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 6d53bb28096df7fb9d7aa7a0d2db7bd3080cafbe available at: https://gist.github.com/8366376bc638f2549670<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4650.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4650.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 134704412ac3f0ac103439834d3c32dd2f3adb94 available at: https://gist.github.com/8366376bc638f2549670<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4650.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4650.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 1058a61ddd09d4bc4cac60b3ca318a188bd63917 available at: https://gist.github.com/8366376bc638f2549670<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4650.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4650.dmg

**Pull request merged**